### PR TITLE
feat: add blog post for 12.3.0.GA

### PR DESCRIPTION
**Pull request by hansemannn**
Related to https://github.com/tidev/titanium-docs/pull/213

### CODE REVIEW

File: posts/2024/sdk_12_3_0_ga.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: 'Titanium SDK 12.3.0.GA released'
+date: '2024-02-16'
+category: 'Release'
+author: 'Hans Knöchel'
+teaser: 'Support for Node.js 20, Apple Vision Pro and new Android camera handling'
+---
+
+The next preview version of the Titanium SDK is now is available: <b>12.3.0.GA</b>. Titanium SDK 12.3.0 is a minor release of the SDK, adding new features and fixing existing bugs:
```
Comment (m1ga) [inline posts/2024/sdk_12_3_0_ga.md:9]:
```suggestion
The next final version of the Titanium SDK is now is available: <b>12.3.0.GA</b>. Titanium SDK 12.3.0 is a minor release of the SDK, adding new features and fixing existing bugs:
```

Comment (cb1kenobi) [reply posts/2024/sdk_12_3_0_ga.md:9]: "Final" doesn't sound right. How about "stable"?

```suggestion
The next stable version of the Titanium SDK is now is available: <b>12.3.0.GA</b>. Titanium SDK 12.3.0 is a minor release of the SDK, adding new features and fixing existing bugs:
```

Comment (m1ga) [reply posts/2024/sdk_12_3_0_ga.md:9]: we use both :) https://tidev.io/blog/sdk_12_2_0_ga -> final, https://tidev.io/blog/sdk_12_2_1_ga -> stable
`stable` might sound better here

Comment (hansemannn) [reply posts/2024/sdk_12_3_0_ga.md]: "Stable" sounds more native English I think! Thanks for noticing!

**Review by ewanharris — APPROVED** [review]

**Review by cb1kenobi — APPROVED** [review]

**Pull request merged**